test/integration/testdata/image-build/test-normal/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
+
+CMD ["/bin/sh", "-c"]
```

Comment:
There are three files with the exact same contents, I'd rather we just have one and modify the tests to point to the same file.

Reply:
Well I think we cannot modify the tests to point to the same file, because the path to the file is exactly the variable in different tests. 
The first file is in a multiple-layer folder , which means '\' will appear in the path
The second file is in a single-layer folder, which means '\' will not appear in the path
The third file is in a special folder with a .dockerignore file. 
We cannot mix them up.

Reply:
Other comments are all accepted.

Reply:
Gotcha, thanks for the explanation